crates/turborepo-lib/src/daemon/server.rs

```diff
@@ -187,13 +188,14 @@ where
             ));
 
         Server::builder()
+            .timeout(Duration::from_millis(100))
```

Comment:
Maybe pull out this duration into a constant labeled `REQUEST_TIMEOUT` or something. The `timeout` doesn't make it super clear what the duration controls.